<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQSystemTrayDialog</class>
 <widget class="QDialog" name="QQSystemTrayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QQTrayIconWidget" name="lianxirenWidget_tray" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QQTrayIconWidget" name="youxiangWidget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QQTrayIconWidget" name="kongjianWidget_tray" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QQTrayIconWidget" name="weiboWidget_tray" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QQTrayIconWidget" name="swiftiyWidget_tray" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="morePButton_tray">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="QQPButton_tray">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQTrayIconWidget</class>
   <extends>QWidget</extends>
   <header location="global">qqtrayiconwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
